Tella/Scenes/Uwazi/Common/TemplateItemView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TemplateItemView.swift
+//  Tella
+//
+//  Created by Gustavo on 03/08/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct TemplateItemView: View {
+    @Binding var template : CollectedTemplate
```

Comment:
I suggest here to pass in parameter a `templateItemViewModel` that contains formatted parameters like template name, isDownloaded, downloadAction, deleteAction and when mapping from CollectedTemplate to  a templateItemViewModel  in the view model you can pass in parameter directly the action located in the viewmodel (downloadAction or deleteAction) so it will be cleaner and easier than to pass the whole `CollectedTemplate`